### DISCUSSION TITLE
Live Search

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#395] [Ideas]
Hi Team,

It will be much helpful if you add auto search. As soon we type in 3 characters you can show results in a drop down and so easy to add to billing.

**Comment by Blair2004** [comment]
Hi, add to billing? Not sure to understand.

**Reply by rsidhaarth** [reply]
When the cashier is on POS, he either has to go to each product or click. Or he has to click on the search option, type the word and then click ok.

Instead, if he clicks on search, and type only 3 letters you can show suggestion like dropdown and so he touches and those products get added.
